src/Model/Contact.php

```diff
@@ -1899,13 +1899,7 @@ public static function getDefaultHeader(array $contact): string
 		switch ($platform) {
 			case 'friendica':
 			case 'friendika':
-				/**
-				 * Picture credits
-				 * @author  Lostinlight <https://mastodon.xyz/@lightone>
-				 * @license CC0 https://creativecommons.org/share-your-work/public-domain/cc0/
-				 * @link    https://gitlab.com/lostinlight/per_aspera_ad_astra/-/blob/master/friendica-404/friendica-promo-bubbles.jpg
-				 */
-				$header = DI::baseUrl() . '/images/friendica-banner.jpg';
+				$header = DI::baseUrl() . '/' . ltrim(DI::config()->get('api', 'mastodon_banner'), '/');
```

Comment:
Wouldn't be an idea to have a central function for fetching the value? I don't really like that the value is fetched and then the ltrim is executed each time at several places.

Reply:
Sure, but I'm not sure where to put it yet.